scripts/test_classify_ci_changes.py

```
@@ -78,6 +87,65 @@ def test_shared_tamarin_policy_input_runs_formal_and_conformance(self) -> None:
+        ]:
+            with self.subTest(path=path):
+                result = classify([path])
+                self.assertTrue(result["run_full"])
+                self.assertTrue(result["run_conformance"])
+                self.assertFalse(result["run_c"])
+                self.assertFalse(result["run_ios"])
+                self.assertFalse(result["run_formal"])
+
+    def test_literal_rust_includes_preserve_their_consumers_lanes(self) -> None:
+        tracked_rust = subprocess.run(
+            ["git", "ls-files", "-z", "--", "*.rs"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)